spiped/dispatch.c

```diff
@@ -98,6 +99,10 @@ callback_conndied(void * cookie)
 	/* We've lost a connection. */
 	A->nconn -= 1;
 
+	/* If only one connection was desired, quit event loop. */
```

Comment:
POV: The dispatcher doesn't know anything about users or command lines or event loops; all it knows is what the call to dispatch_accept told it, i.e., that sometimes we're supposed to set this to a non-zero value when a connection closes.